src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -202,9 +203,21 @@ export class AeternityAdapter extends BaseProtocolAdapter {
     rawAccount: IAccountRaw,
     idx: number,
     globalIdx: number,
-    seed: Uint8Array,
+    seed?: Uint8Array,
   ): IAccount {
-    const wallet = rawAccount.type === ACCOUNT_TYPES.hdWallet
+    if (rawAccount.type === ACCOUNT_TYPES.privateKey && rawAccount.privateKey) {
```

Comment:
Maybe when the type is equal to `privateKey` but the `rawAccount.privateKey` is falsy we should throw an error or return null / do nothing?